sdk/provisioning/Azure.Provisioning/api/Azure.Provisioning.net461.cs

```diff
@@ -0,0 +1,466 @@
+namespace Azure.Provisioning
+{
+    public static partial class CdkExtensions
```

Comment:
We need to rename this as soon as things calm down.

Reply:
I can rename it to ProvisioningExtensions right now.